### PR TITLE
[ubuntu18.04] Update documentation + legacy: solves #2987

### PR DESCRIPTION
**Pull request by dewagter**

### CODE REVIEW

**Comment by dewagter** [comment]
Thanks for the comments @gautierhattenberger
This is all that was needed to still support ubuntu18.04.
I removed 16.04 from the documentation.

File: start.py [hunk 1]
```diff
@@ -21,6 +22,12 @@
 
 import xml.etree.ElementTree
 
+release = lsb_release.get_lsb_information()
```
Comment (gautierhattenberger) [inline start.py:25]: in 20.04, this function doesn't exists. I have a `get_distro_information` that seems to return the same type of information.

**Comment by dewagter** [comment]
Ready to merge. This would also need to go to v6.2.

**Review by gautierhattenberger — APPROVED** [review]
It is also possible to add the check to the `./paparazzi` start script. Should we do that ?

**Pull request merged**